src/js/python/data.js

```diff
@@ -0,0 +1,27 @@
+export default {
+    name: "python",
+    shortName: "py",
+    summary: "Python Script",
+    defaultVersion: "2.7.5",
```

Comment:
1. Do we even need it? `isDefault` below should be enough
2. Should be `"3.8.6"

Reply:
As discussed, `isDefault` is a different concept from `defaultVersion` and as per https://github.com/Exabyte-io/exabyte-web-app/blob/dev/src/exabyte/imports/init_applications/python/application.js#L9 the default is `2.7.5`.